src/com/facebook/buck/android/AndroidLibraryDescription.java

```diff
@@ -145,6 +145,11 @@ public void findDepsForTargetFromConstructorArgs(
 
     Optional<String> getResourceUnionPackage();
 
+    @Value.Default
+    default boolean isSkipPrebuiltRDotJava() {
```

Comment:
Why "prebuilt"?

Reply:
using `prebuilt` as this scenario is only applicable with union package enabled which in turn only leaves us with prebuilt's R Dot Java. setting this would skip the prebuilt's r dot java.

Reply:
I still don't understand what "prebuilt" means in this context.  That's the term we generally use for binaries that are checked into the repository.  Maybe just name this `isSkipNonUnionRDotJava`?  I don't know of any other name for these packages.  `isSkipRDotJavaForThePackageThatDefinesEachResource` doesn't seem quite right. :)

Also, can you document this parameter in the docs subdir?

Reply:
renamed to `isSkipNonUnionRDotJava` and documented the param in docs.